tools/travis_new_boards_check.py

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env python3
+
+import sys
+import os
+import json
+
+import build_board_info
+
+# Get boards in json format
+boards_info_json = build_board_info.get_board_mapping()
+
+# Get all the boards out of the json format
+info_boards = boards_info_json.keys()
+# Turn the dict_keys into a list
+info_boards = list(info_boards)
```

Comment:
I'm late in catching this. There are a few aliased boards returned by `get_board_mapping()`, which are not actual buildable boards:
```
gemma_m0_pycon2018
circuitplayground_express_4h
circuitplayground_express_digikey_pycon2019
```
I drummed up a list comprehension that takes care of not including any aliases:
```python
info_boards = [board for board in boards_info_json.keys() if not boards_info_json[board].get("alias", False)]
```

Reply:
Thanks for catching it, I don't know a lot of python but at the end info_boards will be a list? Because previously it was a dict_key and one line after I turned it into a list, with this new approach, i can remove the `info_boards = list(info_boards)`

Reply:
Yeah, it'll result in a list. List comprehensions are basically the same as this:

```python
>>> foo = [] # or 'list()'
>>> bar = [1, 2, 3]
>>> for item in bar:
	foo.append(item)

>>> foo
[1, 2, 3]
>>> [item for item in bar]
[1, 2, 3]
>>> 
```